src/lib/getKeyDownHandlers.ts

```diff
@@ -0,0 +1,50 @@
+import * as _ from 'lodash'
+import keyboardHandlerFilter from './keyboardHandlerFilter'
+import {
+  IAccessibilityDefinition,
+  AccessibilityActions,
+  ActionsHandler,
+} from 'src/lib/accessibility/interfaces'
+import { IState, IPropsWithVarsAndStyles } from '../../types/theme'
+
+/**
+ * Assigns onKeyDown handler to the Component's part element, based on Component's actions
+ * and keys mappings defined in Accessibility behavior
+ * @param {AccessibilityActions} actions Actions defined in a component.
+ * @param {IAccessibilityDefinition} accessibility The input element which is to loose focus.
+ * @param {IState & IPropsWithVarsAndStyles} props The props which are used to invoke onKeyDown handler passed from top.
+ */
+const getKeyDownHandlers = (
+  actions: AccessibilityActions,
+  accessibility: IAccessibilityDefinition,
+  props: IState & IPropsWithVarsAndStyles,
+): ActionsHandler => {
+  const handlers = {}
+  const actionsDefinition = accessibility.actionsDefinition
```

Comment:
sorry, but still am haunted by thought of renaming `actionsDefinition` part for the following reasons:
- those actions are currently provided for key events - it is better to reflect this in naming
- it seems that we could achieve better consistency if we'll use more specific `actions` <-> `handlers` part

Regarding second point, wouldn't it be better to use `accessibility.keyActions` for defining them, and `accessibility.keyHandlers` for those handlers that were fetched from supported component's actions? This seems to be much more expressive, due to now we do immediately see (just by reading the code) that only key handling concept is handled by accessibility, as well as handlers are populated from general `actionHandlers` defined by component (this 'general' part is necessary, due to these component's action handlers could be consumed later to map actions from other sources, like joystick or whatever :)